game/weather.py

```diff
@@ -158,6 +158,8 @@ def generate_wind(self) -> WindConditions:
 class Conditions:
     time_of_day: TimeOfDay
     start_time: datetime.datetime
+    pressure: float
+    temperature: float
```

Comment:
```suggestion

    #: Pressure at sea level in inches of mercury.
    pressure: float
    
    #: Temperature at sea level in Celcius.
    temperature: float

```

Should these be part of `Weather` so you can have different ranges based on rain/storm/etc?

Reply:
Yes, good idea. Will change.